riscv-sbi.adoc

```diff
@@ -1674,6 +1675,259 @@ The possible error codes returned in `sbiret.error` are shown in the
 | sbi_pmu_counter_fw_read         | 0.3         | 5   | 0x504D55
 |===
 
+== Attestation and Measurement Extension (EID #0x41545354 "ATST")
+
+The Attestation and Measurement (ATST) Extension provides a set of functions for
+implementing TCB attestation flows and for extending TCB measurements. The SBI
+implementation composes those measurements into an attestation evidence that can
+be used by attestation verifiers, either locally or remotely.
+
+These functions provide the capabilities and features required to implement
+confidential computing, and also  measured and verified boot.
```

Comment:
These functions provide the capabilities and features required to implement attestation for usages such as, confidential computing, measured boot and verified boot.